.github/workflows/twitter-together.yml

```diff
@@ -6,18 +6,18 @@ jobs:
     runs-on: ubuntu-latest
     if: github.event_name == 'pull_request'
     steps:
-      - uses: gr2m/twitter-together@v1.x
+      - uses: twitter-together/action@v2
```

Comment:
We should pin this action to an exact SHA

Reply:
@MarshallOfSound Fixed.